src/main/java/org/folio/dew/batch/acquisitions/edifact/jobs/EdifactExportJobConfig.java

```diff
@@ -22,31 +29,36 @@ public class EdifactExportJobConfig {
 
   private Job constructEdifactExportJob(JobBuilder jobBuilder,
                                         EdiExportJobCompletionListener ediExportJobCompletionListener,
-                                        Step mapToEdifactOrdersStep,
-                                        Step saveToFTPStep,
+                                        Step mapToFileStep,
                                         Step saveToMinIOStep,
-                                        Step createExportHistoryRecordsStep) {
+                                        Step saveToFTPStep,
+                                        Step createExportHistoryRecordsStep,
+                                        Map<Step, JobExecutionDecider> optionalStepDeciders) {
     return jobBuilder.incrementer(new RunIdIncrementer())
       .listener(ediExportJobCompletionListener)
-      .start(mapToEdifactOrdersStep)
+      .start(mapToFileStep)
       .next(saveToMinIOStep)
-      .next(saveToFTPStep)
-      .next(createExportHistoryRecordsStep)
-      .build();
+      .next(optionalStepDeciders.get(saveToFTPStep))
```

Comment:
we should always have an ability to save to ftp even for claiming type, so decider here is not needed

Reply:
I saw that TransimitionMethod has also File Download option, so please ignore my commnet
![image](https://github.com/user-attachments/assets/ada82fb6-7b7c-461d-b6f4-a1a1c559457b)